project/CopyrightHeader.scala

```diff
@@ -68,10 +66,8 @@ trait CopyrightHeader extends AutoPlugin {
     }
 
     override def apply(text: String, existingText: Option[String]): String = {
-      existingText
-        .map(updateLightbendHeader)
-        .getOrElse(commentCreator(text, existingText))
-        .trim
+      val updatedExistingText = existingText.map(updateLightbendHeader)
```

Comment:
The change here is not right:(, The original one is OK but the existing text is not.